packages/account/src/Sections/Security/ApiToken/api-token.tsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
 import classNames from 'classnames';
 import { Formik, Form, Field, FormikErrors, FieldProps, FormikHelpers } from 'formik';
-import { useApiToken } from '@deriv/api';
```

Comment:
Are we removing `useApiToken` hook completely or will it be part of another PR?

Reply:
We are removing the api hooks entirely for now.